src/test/java/io/gingersnapproject/MySQLResources.java

```diff
@@ -0,0 +1,54 @@
+package io.gingersnapproject;
+
+import java.util.Map;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.Base58;
+import org.testcontainers.utility.MountableFile;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class MySQLResources implements QuarkusTestResourceLifecycleManager {
+
+   private GenericContainer<?> cacheManager;
+   private MySQLContainer<?> db;
+
+   @Override
+   public Map<String, String> start() {
+      cacheManager = new GenericContainer<>("quay.io/gingersnap/cache-manager")
+            .withNetworkAliases("infinispan-" + Base58.randomString(6))
+            .withExposedPorts(8080, 11222)
```

Comment:
Maybe we'll have to map 8081 to 8080, so the db-syncer starts using 8080

Reply:
I don't think that's necessary as TestContainers already exposes this container port differently to what we expose. If you look in the logs you will see something like:

```
2022-12-09 09:28:33,909 INFO  [org.tes.con.wai.str.HttpWaitStrategy] (pool-3-thread-1) /unruffled_swanson: Waiting for 60 seconds for URL: http://localhost:49157/q/health (where port 49157 maps to container port 8080)
```

and when configuring the db-syncer, we use the container mapping for the actual port value:

```java
"gingersnap.cache.uri", String.format("hotrod://%s:%d", cacheManager.getHost(), cacheManager.getMappedPort(11222)),
```